arweave/caip10.md

```diff
@@ -0,0 +1,49 @@
+---
+namespace-identifier: arweave-caip10
+title: Arweave Namespace - Addresses
+author: Rohit Pathare (@ropats16), Phil Billingsby (@pbillingsby)
+discussions-to:
+status: Draft
+type: Standard
+created: 2022-09-01
+requires: ["CAIP-2", "CAIP-10"]
+---
+
+# CAIP-10
+
+*For context, see the [CAIP-10][] specification.*
+
+## Abstract
+In CAIP-10 an account identification scheme is defined. This is the implementation of CAIP-10 for Arweave network.
+
+## Rationale
+
+Particularities of syntax for Arweave "accounts" have been specified.  
+
+## Syntax
+
+Arweave addresses have a normalization of lowercase letters, uppercase letters, numbers and `-`s.
+
+### Backwards Compatibility
+
+Not applicable.
+
+## Test Cases
+
+```
+# Arweave mainnet
+ar:1:kY9RAgTJEImkBpiKgVeXrsGV02T-D4dI3ZvSpnn7HSk
```

Comment:
Can you mention somewhere what format the address is encoded with? Is it base64?

Reply:
base64Url encoded 😎

Reply:
Updated the PR to reflect the format in which the addresses are encoded.